Jenkinsfile

```diff
@@ -59,7 +59,7 @@ def buildJob(Map conf, compiler){
         def env4make = conf.get("env4make", "")
         def image = conf.get("image", "miopen")
         def cmd = conf.get("cmd", "")
-        def gpu_arch = conf.get("gpu_arch", "all")
+        def gpu_arch = conf.get("gpu_arch", "gfx900;gfx906")
```

Comment:
I believe this flag is for MIOpenTensile, which will still treat this as `all`. This is because tensile only allows setting one arch or all archs. So when setting the archs to `gfx900;gfx906` internally it will set the tensile arch to "all":

https://github.com/ROCmSoftwarePlatform/MIOpenTensile/blob/develop/CMakeLists.txt#L37

Reply:
Thanks.

Reply:
I think we can tolerate that provided that dockers images will not be rebuilt too often. However it would be nice to save some time during build of Tensile, in the near future...

Reply:
@pfultz2 btw in `hip-clang.docker` we are passing `-DAMDGPU_TARGETS=${GPU_ARCH}` to Tensile build, so `Tensile_ARCHITECTURE`  [here](https://github.com/ROCmSoftwarePlatform/MIOpenTensile/blob/develop/CMakeLists.txt#L37) should take its value, right?

Reply:
[Side note] I think that `if(AMDGPU_TARGETS MATCHES ";")` takes true path only if `AMDGPU_TARGETS` equal to `";"`. When it contains `gfx900;gfx906`, `'if(...)` takes false path, right? -- https://cmake.org/cmake/help/latest/command/if.html#matches

Reply:
> I think we can tolerate that provided that dockers images will not be rebuilt too often. However it would be nice to save some time during build of Tensile, in the near future...

Yea, I think we need to pull down prebuilt binaries. Or we could have a jenkins job that builds the binaries if they haven't been built already, and then we can archive it on jenkins. 

> btw in hip-clang.docker we are passing -DAMDGPU_TARGETS=${GPU_ARCH} to Tensile build, so Tensile_ARCHITECTURE here should take its value, right?

Yes, but if it is `gfx900;gfx906` then it will still build for `all`.

>  I think that if(AMDGPU_TARGETS MATCHES ";") takes true path only if AMDGPU_TARGETS equal to ";". if("gfx900;gfx906" MATCHES ";") takes false path, isn't it?

No, this is a regex match. If you want it to be true for only `;` then you would do `if(AMDGPU_TARGETS STREQUAL ";")`.

Reply:
Of course, I dont think this change is a blocker, so I approved it already.

Reply:
> No, this is a regex match. If you want it to be true for only `;` then you would do `if(AMDGPU_TARGETS STREQUAL ";")`.

Aha, it works like `echo "a;b" | grep ";"`. Thanks for reviewing!